src/main/scala/no/ndla/conceptapi/model/api/UpdatedConcept.scala

```diff
@@ -16,8 +16,9 @@ case class UpdatedConcept(
     @(ApiModelProperty @field)(description = "The language of this concept") language: String,
     @(ApiModelProperty @field)(description = "Available titles for the concept") title: Option[String],
     @(ApiModelProperty @field)(description = "The content of the concept") content: Option[String],
-    @(ApiModelProperty @field)(description = "An image-api ID for the concept meta image") metaImage: Option[
-      NewConceptMetaImage],
+    @(ApiModelProperty @field)(description = "An image-api ID for the concept meta image") metaImage: Either[
+      Null,
+      Option[NewConceptMetaImage]], // We use this type to make json4s understand the difference between null and missing fields
```

Comment:
Om vi skal gjøre dette flere enn 1 sted og kommentere det hver gang så stemmer jeg for at vi lager en type for det, feks i pakke objektet og kommenterer det der.
```suggestion
    @(ApiModelProperty @field)(description = "An image-api ID for the concept meta image") metaImage: Deletable[NewConceptMetaImage],
```
Så kan du definere typen i pakke objektet e.l:
`type Deletable[T] = Either[Null, Option[T]]`